aio/content/marketing/resources.json

```
@@ -230,11 +230,11 @@
             "title": "Angular CLI",
             "url": "https://cli.angular.io"
           },
-          "d1": {
-            "desc": "Static analysis for Angular projects.",
+          "angular-eslint": {
+            "desc": "The tooling which enables ESLint to lint Angular projects.",
             "logo": "",
-            "title": "Codelyzer",
-            "url": "https://github.com/mgechev/codelyzer"
+            "title": "Angular ESLint",
+            "url": "https://github.com/angular-eslint/angular-eslint"
```

Suggestion:
Personally, I think it is better to have the browser automatically scroll to the README (by appending `#readme` to the URL), so people can read about the project and how to get started:

```suggestion
            "url": "https://github.com/angular-eslint/angular-eslint#readme"
```

WDYT?

---

Thanks. That's a good suggestion. I committed the changes.